hexstody-btc-test/src/runner.rs

```diff
@@ -249,9 +333,10 @@ async fn setup_api_regtest(
     });
     tokio::spawn({
         let client = make_client();
-        let polling_duration = Duration::from_secs(30);
+        let polling_duration = Duration::from_millis(100);
```

Comment:
Regression, that function should not poll so fast

Reply:
Resolved: f121443e6a9d7086cae7fff1aa9a0efa5bcbe35c